### PR TITLE
Subscriptions: metering fix

### PR DESCRIPTION
**Pull request by prateekbh**
- recognizes metering response from Viewer

### CODE REVIEW

File: extensions/amp-subscriptions/0.1/entitlement.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class Entitlement {
     this.subscriptionToken = subscriptionToken;
     /** @const {boolean} */
     this.loggedIn = loggedIn;
-    /** @const {?MeteringData} */
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/0.1/entitlement.js]: Add `@type` instead then. But why is it not `@const` anymore?

File: extensions/amp-subscriptions/0.1/viewer-subscription-platform.js [hunk 2]
```diff
@@ -130,6 +130,17 @@ export class ViewerSubscriptionPlatform {
             break;
           }
         }
+      } else if (decodedData['metering'] && !decodedData['entitlements']) { // No entitlements
+        dev().assert(this.currentProductId_, 'Current product is not set');
+        entitlement = new Entitlement({
+          source: '',
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/0.1/viewer-subscription-platform.js]: Use `decodedData['iss']` here.

File: extensions/amp-subscriptions/0.1/viewer-subscription-platform.js [hunk 3]
```diff
@@ -130,6 +130,17 @@ export class ViewerSubscriptionPlatform {
...
+          raw: '',
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/0.1/viewer-subscription-platform.js]: `raw == the jwt string`

Comment (prateekbh) [reply extensions/amp-subscriptions/0.1/viewer-subscription-platform.js]: Done

File: extensions/amp-subscriptions/0.1/viewer-subscription-platform.js (hunk 2 above)
Comment (prateekbh) [reply extensions/amp-subscriptions/0.1/viewer-subscription-platform.js]: Done

File: extensions/amp-subscriptions/0.1/entitlement.js (hunk 1 above)
Comment (prateekbh) [reply extensions/amp-subscriptions/0.1/entitlement.js]: Done

**Pull request merged**